new/LIT6224MGBebaChurch.xml

```diff
@@ -0,0 +1,109 @@
+                <editor key="SH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine
+                    multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Hagiography"/>
+                    <term key="Miracle"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="SH" when="2020-11-06">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+
+                <listBibl type="editions">
+                    <bibl>
+                        <ptr target="bm:Arras1953GeorgiusText"/>
+                        <citedRange unit="page">1–4 (14th miracle)</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:Arras1953GeorgiusTrans"/>
+                        <citedRange unit="page">1–3 (14th miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+
+                <listBibl type="secondary">
+                    <bibl>
+                        <ptr target="bm:Strelcyn1974CatJRylands"/>
+                        <citedRange unit="page">34</citedRange>
+                        <citedRange unit="item">Ms 12, II.15 (14th miracle)</citedRange>
+                    </bibl>
+                    <bibl>
+                        <ptr target="bm:GrebTiss1935Codices"/>
+                        <citedRange unit="page">719</citedRange>
+                        <citedRange unit="item">Vat 233, 3.15 (14th miracle)</citedRange>
+                    </bibl>
+                </listBibl>
+
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT6224MGBebaChurch"
```

Comment:
Aha, now I see where the attestation relations in the other files came from :) These are redundant, you don't need to waste your time and go on deleting the existing ones, but no need to add them in the future. I imaging that your inspiration came from the Miracles of Mary files, which often feature this relation: They refer there to EMML manuscripts which are not in the process of being encoded, so this was the easiest way of recording the attestation. Here, since the manuscript is encoded, it is not necessary.